bundles/org.openhab.binding.heliosventilation/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,235 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heliosventilation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Ventilation Thing Type -->
+	<thing-type id="ventilation">
+		<label>HeliosVentilation (KWL)</label>
+		<description>A domestic ventilation system (KWL) from Helios.</description>
+
+		<channels>
+            <channel id="outsideTemp" typeId="outside_temperature" />
```

Comment:
formatting looks strange. XML are not automatically formatted on save due to restrictions in Eclipse. Use Ctrl-Shift-F.